ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 set -ex
 
+pushd /tmp
+wget https://github.com/Kitware/CMake/releases/download/v3.30.5/cmake-3.30.5-linux-x86_64.tar.gz
```

Comment:
Looks like vv3.30.5 is no longer available. Could you please use https://github.com/Kitware/CMake/releases/download/v3.30.6/cmake-3.30.6-linux-x86_64.tar.gz

Reply:
I strongly suggest using the common utility functions to retrieve the version from `versions.json`, verify your download, and write the component version used to the `component_versions.txt` so that it's stored in the image for reference. 

Most files have examples of doing this, but `https://github.com/Azure/azhpc-images/blob/master/common/install_azcopy.sh` is fairly simple. This should be done for all components that are being installed.